src/server/task-list/controller.test.js

```diff
@@ -72,6 +72,17 @@ describe('#taskListController', () => {
   })
 
   it('Should return the correct task list items when there is state enough to show all sections', async () => {
```

Comment:
are we removing the test that covers the scenario where all the sections are visible? (that's what the test description says so I guess we need another test or worst case we need to rename this one)